lib/vanagon/platform.rb

```diff
@@ -101,6 +101,10 @@ def initialize(name)
       @ssh_port = 22
       @provisioning = []
       @install ||= "install"
+      @target_user ||= "root"
+      @find ||= "find"
+      @sort ||= "sort"
+      @copy ||= "cp"
```

Comment:
Out of scope for a merge or even a casual PR but as the number of built-in tools we rely on grows, we're probably going to have subclass platform utilities.